advanced-achievements-plugin/src/main/resources/config.yml

```diff
@@ -188,6 +188,12 @@ AllAchievementsReceivedRewards:
   IncreaseMaxOxygen: 30
   Experience: 100
 
+# Title shown on the root advancement.
```

Comment:
Let's group these with the two existing advancement-related parameters in _config.yml_. 😄